spec/book_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec'
```

Comment:
- [ ] Kindly fix those failed tests as now when I run the tests I get 14 failures.
![image](https://user-images.githubusercontent.com/24703171/233598247-294739af-8716-47db-bb3e-1340d99d6c36.png)